### PR TITLE
Strategic council

### PR DESCRIPTION
**Pull request by bvhme**
I propose we add a strategic council to support our work.

This is not named 'advisory' because there is a statutory advisory council that has formal obligations I prefer not to see this body be required to fulfil.

I would love suggestions and questions to this pull request.

-----
[View rendered organization/index.md](https://github.com/publiccodenet/about/blob/strategic-council/organization/index.md)
[View rendered organization/strategic-council.md](https://github.com/publiccodenet/about/blob/strategic-council/organization/strategic-council.md)

### CODE REVIEW

File: organization/strategic-council.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+Membership of the strategic council is on invitation from the board of directors.
+
+Members of the strategic council are expected to:
+
+* advise to the board of directors or general assembly on strategic matters
+* serve as a public ambassador for the mission and organization
+* follow along with the briefings sent by the board of directors
+* participate in (email) discussions of the council
+
+This commitment is estimated to be about 15 minutes per week.
```
Comment (ericherman) [inline organization/strategic-council.md:20]: I expect the estimate is expected to be an _average_ weekly amount; if so, might be better to be explicit about that.

**Review by ericherman — APPROVED** [review]
A sound idea, and this is a fine start.

Can go in as-is or with or without the "average" word added to time commitment.

File: organization/strategic-council.md (hunk 1 above)
Comment (bvhme) [reply organization/strategic-council.md]: Added.

**Comment by Ainali** [comment]
Membership should perhaps be expanded a bit. It appears that an invitation is enough, it doesn't seem to matter if the invitee accepts it or not. We should also be explicit on how membership can be terminated.

**Comment by bvhme** [comment]
@Ainali I've tried to reflect your comments

**Comment by Ainali** [comment]
> This is not named 'advisory' because there is a statutory advisory council that has formal obligations I prefer not to see this body be required to fulfil.

This is a good idea and got me thinking if we can find another term than 'advisors' to try to avoid future confusions where people from the different groups get mixed up? Perhaps 'strategists' or 'counsellors' for people here?

**Comment by Ainali** [comment]
One question I get is *how* the council advise the board? Do the strategic council members have to come to an agreement on the best advice to give or can individual members give the board different, perhaps even competing, views on an issue? I guess my question boils down to, is the council acting as a collective or are the members 'free agents' with a common label?

**Comment by bvhme** [comment]
@Ainali At this time I'd argue for not defining this yet. Hopefully, there is natural discussion that will feed recommendation. I'd love for those recommendations to provide conflicting viewpoints when those are available.

**Review by ericherman — APPROVED** [review]

**Comment by bvhme** [comment]
Please hold of on merging this. A (specific) new contributor would like a chance to contribute before we merge it.

File: organization/strategic-council.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+The strategic council is a group of individual advisors that helps the Foundation For Public Code [board of directors](../organization/governance-model.md#board-of-directors) with key strategic descisions as well as serve as ambasadors of it's [mission](mission.md) and organization.
+
+## Membership of the strategic council
+
+Potential new members of the strategic council are invited by the board of directors.
+On acceptance of the role the member is added to the strategic council.
+
+Members of the strategic council are expected to:
+
+* advise to the board of directors or general assembly on strategic matters
```
Comment (ElenaFdR) [inline organization/strategic-council.md:16]:
```suggestion
* advise the board of directors or general assembly on strategic matters
```

File: organization/strategic-council.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+* serve as a public ambassador for the mission and organization
```
Comment (ElenaFdR) [inline organization/strategic-council.md:17]:
```suggestion
* serve as a public ambassador for the organization and its mission
```

File: organization/strategic-council.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+* follow along with the briefings sent by the board of directors
```
Comment (ElenaFdR) [inline organization/strategic-council.md:18]:
```suggestion
* follow along with briefings sent by the board of directors
```

File: organization/strategic-council.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+* participate in (email) discussions of the council
```
Comment (ElenaFdR) [inline organization/strategic-council.md:19]:
```suggestion
* participate in the council's (email) discussions 
```

File: organization/strategic-council.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+This commitment is estimated to be about an average 15 minutes per week.
```
Comment (ElenaFdR) [inline organization/strategic-council.md:21]:
```suggestion
This commitment is estimated to be an average of 15 minutes per week.
```

File: organization/strategic-council.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+This is a voluntary role and non-remunerated.
+Membership of the strategic council can be terminated by the member of the strategic council or the board of directors and at any time in writing.
```
Comment (ElenaFdR) [inline organization/strategic-council.md:23]:
```suggestion
Membership of the strategic council can be terminated at any time in writing by either the member of the strategic council or the board of directors.
```

**Review by ElenaFdR — COMMENTED** [review]
Lots of small stylistic suggestions.

File: organization/strategic-council.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+---
+
+# Strategic council
+
+The strategic council is a group of individual advisors that helps the Foundation For Public Code [board of directors](../organization/governance-model.md#board-of-directors) with key strategic descisions as well as serve as ambasadors of it's [mission](mission.md) and organization.
+
+## Membership of the strategic council
+
+Potential new members of the strategic council are invited by the board of directors.
+On acceptance of the role the member is added to the strategic council.
```
[the remark this answers is not in the document]
Comment (bvhme) [reply organization/strategic-council.md:12]: Here I tried to reflect that basically the role, as set out in this resource, is what also needs to be accepted, not just the invitation.

File: organization/strategic-council.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+
+The board of directors invites new members to join the strategic council.
+On acceptance of the role the member is added to the strategic council.
+
+Members of the strategic council are expected to:
+
+* advise the board of directors or general assembly on strategic matters
+* serve as a public ambassador for the organization and its mission
+* follow along with briefings sent by the board of directors
+* participate in the council's (email) discussions
```
Comment (bvhme) [inline organization/strategic-council.md:19]: Remove trailing space

```suggestion
* participate in the council's (email) discussions
```

**Review by ericherman — DISMISSED** [review]

**Comment by ElenaFdR** [comment]
The creation of the strategic council will require other pages to be updated to mention it, including [the governance model](https://about.publiccode.net/organization/governance-model.html).

**Comment by ElenaFdR** [comment]
We're missing info on how to become a strategic council member (what the appointments process is). I've already had one expression of curiosity from someone, and have promised to add it. As a start, I propose this very lightweight process:

- we approach people OR people can email us to apply
- Foundation staff (or the board?) have an initial discussion to confirm mutual understanding of the role/interest
- board or GA approve (I'd have to reread the Articles of Association to guess where this responsibility should lie)
- Voila! we have a new strategic council member

**Review by ericherman — APPROVED** [review]

**Review by ericherman — APPROVED** [review]

**Pull request merged**